### DISCUSSION TITLE
TFT 35 V3 - Touchpad stopped working after updating the firmware

### DISCUSSION THREAD

**Discussion by Redxx** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1622] [Q&A] [answered]
Hi, 
the touchpad stopped working after updating the firmware, 
TFT 35 v3, now when I touch the screen it lights up, but the touch panel does not work.
In marlin mode it works perfectly, but not in touch mode. any ideas?
sorry for my bad english.

**Comment by kisslorand** [comment, marked as answer]
Put an empty "Reset.txt" file on your SD card.

**Reply by Redxx** [reply]
solved, it worked, thank you

**Reply by kisslorand** [reply]
Great. Now please close this ticket.

**Reply by sarvenn** [reply]
@kisslorand does the reset button on the tft do the same job with that txt?

**Comment by kisslorand** [comment]
No, that button just makes the TFT to reboot.

**Reply by sarvenn** [reply]
What does reset.txt do on the backround?

**Comment by kisslorand** [comment]
Nothing. It as active only at TFT startup. It resets the TFT touch calibration and starts a new calibration procedure.
